lib/tests/missing-zod-chains.test.ts

```diff
@@ -66,7 +66,7 @@ test("missing-zod-chains", async () => {
         .passthrough();
       const nulltype = z.object({}).partial().passthrough();
       const anyOfType = z.union([
-        z.object({}).partial().passthrough(),
+        z.object({}).partial().passthrough().nullable(),
```

Comment:
Same here. On line 21 of this file, the `anyOf` type has `nullable: true`, so this test was asserting a lack of the support being added in this PR.